### DISCUSSION TITLE
Artwork filename customizable

### DISCUSSION THREAD

**Discussion by nmycps** [Marekkon5/onetagger#421] [Ideas]
Since 'cover.jpg' is the standard filename, I would like to have the option to auto-name the file based on 'Artist - Album.jpg'. Thanks for consideration.

**Comment by Marekkon5** [comment]
Added in latest commit, template tags are same as renamer. You can get the binary from actions tab. Feel free to comment or reopen if you have more issues / questions.

**Comment by nmycps** [comment]
Hello. I got the latest commit. About that update, I'm looking for it but don't see it. Is the file renaming option supposed to be in 'Save album art to file' when you switch it on? 
Thanks!

**Reply by Marekkon5** [reply]
It should be down lower in Advanced settings

**Comment by nmycps** [comment]
That is really great! Thank you!
